index.html

```diff
@@ -110,7 +110,43 @@ <h2>A project for first-time contributions</h2>
             <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
             <!-- make sure there is one line of space above and below your card -->
             <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+            
+            <!-- ________ Suraj's card START ________  -->
+            <div class="card">
+                <p class="name">Suraj Sarangi</p>
+                <p class="contact">
+                    <i class="fab fa-github"></i>
+                    <a href="http://github.com/SurajSarangi" target="_blank">SurajSarangi</a>
+                </p>
+                <p class="about">
+                    Engineer || Front-end dev || Back-end dev || Data scientist
+                    <br><br>
+                    Find me on <a href="https://twitter.com/_suraj_sarangi" target="_blank">Twitter</a> or <a href="http://linkedin.com/in/surajsarangi" target="_blank">LinkedIN</a>
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
 
```

Comment:
Hello @SurajSarangi, thank you for contributing. There are some changes that need to be made to your index.html file. Please remove this blank line and be sure to run `npm run prettier-html` to insure that your file meets the styling requirements. Thank you.